src/RemoteTech/Modules/ModuleRTDataTransmitter.cs

```diff
@@ -91,7 +91,7 @@ private IEnumerator Transmit(Callback callback = null)
                 }
```

Comment:
In FancyMouse's fix, line 75-77 changed to:

`               scienceDataQueue.RemoveAt(0);`
`               scienceData.triggered = true;`
`               var subject = ResearchAndDevelopment.GetSubjectByID(scienceData.subjectID);`
`               if (subject == null)`
`                   subject = new ScienceSubject("", "", 1, 0, 0);`
`               int packets = Mathf.CeilToInt(scienceData.dataAmount / PacketSize);`

This seems to be the quirky NRE hack they were talking about.